code/js/controllers/PlexController.js

```diff
@@ -5,13 +5,13 @@
 
   var controller = new MouseEventController({
     siteName: "Plex.tv",
-    play: "button[aria-label='Play'] > i",
-    pause: "button[aria-label='Pause'] > i",
-    playNext: "button[aria-label='Next'] > i",
-    playPrev: "button[aria-label='Previous'] > i",
-    mute: "button[aria-label='Mute Volume'] > i",
+    play: "i[class^=plex-icon-player-play-]",
+    pause: "i[class^=plex-icon-player-pause-]",
+    playNext: "i[class^=plex-icon-player-next-]",
+    playPrev: "i[class^=plex-icon-player-prev-]",
+    mute: "i[class^=plex-icon-player-volume-]",
 
-    song: "[class^=' MetadataPosterTitle-title'] [role=link]",
+    song: "title",
```

Comment:
This isn't always true. You can now collapse the player to the bottom of the window and navigate your library. I'd keep the prior selector, as it shouldn't change with language.

Reply:
From my observations, while something is playing, it does stay true. If nothing was started, it shows "Plex".

The previous selector is not specific enough. It can select elements of the home page, like an episode from the "On Deck" part. It is also shows only season when playing a TV episode, only the singer when playing music and nothing when playing a movie.

 The only downside to using <title> is the "▶ " characters. We can find something better than <title>, but the selector I replaced was certainly broken.